docs/client_reference.rst

```diff
@@ -0,0 +1,261 @@
+.. _aiohttp-client-reference:
+
+HTTP Client Reference
+=====================
+
+.. highlight:: python
+
+.. module:: aiohttp.web
+
+
+.. _aiohttp-web-request:
+
+
+request function
+----------------
+
+.. py:function:: request(method, url, *, params=None, data=None, headers=None, cookies=None, files=None, auth=None, allow_redirects=True, max_redirects=10, encoding='utf-8', version=HttpVersion(major=1, minor=1), compress=None, chunked=None, expect100=False, connector=None, loop=None, read_until_eof=True, request_class=None, response_class=None, test=None)
+   :module: aiohttp.client
+
+   Performs an asyncronous http request. Returns a response object.
+
+   The function is a :ref:`coroutine <coroutine>`.
+
+   :param str method: HTTP method
+
+   :param str url: Request URL
+
+   :param params: Dictionary or bytes to be sent in the query
+                  string of the new request (optional)
+
+   :param data: Dictionary, bytes, or file-like object to
+                send in the body of the request (optional)
+
+   :param dict headers: Dictionary of HTTP Headers to send with
+                        the request (optional)
+
+   :param dict cookies: Cookies to send with the request (optional)
+
+   :param aiohttp.helpers.BasicAuth auth: BasicAuth named tuple that represents
+                                          HTTP Basic Auth (optional)
+
+   :param bool allow_redirects: If set to ``False``, do not follow redirects.
+                                ``True`` by default (optional).
+
+   :param aiohttp.protocol.HttpVersion version: Request http version (optional)
+
+   :param bool compress: Set to ``True`` if request has to be compressed
+                         with deflate encoding.
+                         ``None`` by default (optional).
+
+   :param chunked: Set to chunk size for chunked transfer encoding.
```

Comment:
If here I can set chunk size, shouldn't it declare int type? Or floats are ok? (: